MapboxNavigation/ManeuverView.swift

```diff
@@ -120,7 +120,7 @@ public class ManeuverView: UIView {
                 flip = true
             case .uTurn:
                 ManeuversStyleKit.drawArrow180right(frame: bounds, resizing: resizing, primaryColor: primaryColor)
-                flip = angle < 0
+                flip = step.drivingSide == .right
```

Comment:
Actually, should this be .left. Looking elsewhere:

https://github.com/mapbox/mapbox-navigation-ios/blob/eb5b231c8d9b6fa4a4c46ed222457d725b2963e5/MapboxNavigation/ManeuverView.swift#L89

Reply:
`ManeuversStyleKit.drawArrow180right(frame:resizing:primaryColor:)` draws an arrow to the right by default, right? So we’d want to flip that when driving to the right (U-turning to the left) but keep it as-is when driving to the left (U-turning to the right).